spec/features/readers/index_spec.rb

```diff
@@ -31,11 +33,34 @@
     describe"US6 Parent Index sorted by Most Recently Created" do
       it "I visit the parent index, I see that records are ordered by most recently created first and next to each of the records I see when it was created" do
         visit "/readers"
-        save_and_open_page
 
         expect(amy.name).to appear_before(todd.name)
         expect(vivian.name).to appear_before(amy.name)
         expect(todd.name).to appear_before(penny.name)
+      end
+    end
+
+    describe "US11 Parent Creation" do
+      it "I see a link to create a new Parent record, 'New Parent' When I click this link
+      Then I am taken to '/parents/new' where I  see a form for a new parent record" do
+      visit "/readers"
+
+        expect(page).to have_link("New Reader")
+        click_link("New Reader")
+        expect(current_path).to eq("/readers/new")
+      end
+      
+      it " I fill out the form with a new parent's attributes, And I click the button 'Create Reader' to submit the form
```

Comment:
Tests for form that tests on index page that a link is seen, then its clicked and where its routed to.  Then the new page is routed to a form that is filled out and the create button is clicked.  Then it is routed back to the original index page with updated reader info